2024-02-12 17:00:32,082 INFO    MainThread:217525 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-12 17:00:32,082 INFO    MainThread:217525 [wandb_setup.py:_flush():76] Configure stats pid to 217525
2024-02-12 17:00:32,082 INFO    MainThread:217525 [wandb_setup.py:_flush():76] Loading settings from /u3/oqcardos/.config/wandb/settings
2024-02-12 17:00:32,082 INFO    MainThread:217525 [wandb_setup.py:_flush():76] Loading settings from /u3/oqcardos/motion_prediction/per-motion-prediction/wandb/settings
2024-02-12 17:00:32,082 INFO    MainThread:217525 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-12 17:00:32,082 INFO    MainThread:217525 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-02-12 17:00:32,083 INFO    MainThread:217525 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-12 17:00:32,083 INFO    MainThread:217525 [wandb_init.py:_log_setup():526] Logging user logs to /u3/oqcardos/motion_prediction/per-motion-prediction/wandb/run-20240212_170032-aljcz9fe/logs/debug.log
2024-02-12 17:00:32,083 INFO    MainThread:217525 [wandb_init.py:_log_setup():527] Logging internal logs to /u3/oqcardos/motion_prediction/per-motion-prediction/wandb/run-20240212_170032-aljcz9fe/logs/debug-internal.log
2024-02-12 17:00:32,083 INFO    MainThread:217525 [wandb_init.py:init():566] calling init triggers
2024-02-12 17:00:32,083 INFO    MainThread:217525 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'optimizer': 'AdamW', 'lr': 5e-05, 'usingScheduler': 'linear', 'dataset': 'single-supports', 'epochs': 100}
2024-02-12 17:00:32,083 INFO    MainThread:217525 [wandb_init.py:init():591] re-initializing run, found existing run on stack: 0cwy31kt
2024-02-12 17:00:32,085 INFO    MainThread:217525 [wandb_run.py:_finish():1970] finishing run oqcardoso/LLM_TOTURIAL/0cwy31kt
2024-02-12 17:00:32,086 ERROR   MainThread:217525 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2024-02-12 17:00:32,086 INFO    MainThread:217525 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-12 17:00:32,086 INFO    MainThread:217525 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-02-12 17:00:32,086 INFO    MainThread:217525 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-02-12 17:00:32,086 INFO    MainThread:217525 [wandb_run.py:_restore():2193] restore
2024-02-12 17:00:32,086 INFO    MainThread:217525 [wandb_run.py:_restore():2199] restore done
2024-02-12 17:00:37,848 INFO    MainThread:217525 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-02-12 17:00:37,849 INFO    MainThread:217525 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-02-12 17:00:37,856 INFO    MainThread:217525 [wandb_run.py:_footer_sync_info():3825] logging synced files
2024-02-12 17:00:37,933 INFO    MainThread:217525 [wandb_init.py:init():616] starting backend
2024-02-12 17:00:37,933 INFO    MainThread:217525 [wandb_init.py:init():620] setting up manager
2024-02-12 17:00:37,935 INFO    MainThread:217525 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-02-12 17:00:37,935 INFO    MainThread:217525 [wandb_init.py:init():628] backend started and connected
2024-02-12 17:00:37,941 INFO    MainThread:217525 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-02-12 17:00:37,941 INFO    MainThread:217525 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-02-12 17:00:37,941 INFO    MainThread:217525 [wandb_init.py:init():720] updated telemetry
2024-02-12 17:00:37,954 INFO    MainThread:217525 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-02-12 17:00:39,075 INFO    MainThread:217525 [wandb_run.py:_on_init():2262] communicating current version
2024-02-12 17:00:39,118 INFO    MainThread:217525 [wandb_run.py:_on_init():2271] got version response 
2024-02-12 17:00:39,118 INFO    MainThread:217525 [wandb_init.py:init():804] starting run threads in backend
2024-02-12 17:00:43,711 INFO    MainThread:217525 [wandb_run.py:_console_start():2241] atexit reg
2024-02-12 17:00:43,711 INFO    MainThread:217525 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-02-12 17:00:43,711 INFO    MainThread:217525 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-02-12 17:00:43,711 INFO    MainThread:217525 [wandb_run.py:_redirect():2186] Redirects installed.
2024-02-12 17:00:43,712 INFO    MainThread:217525 [wandb_init.py:init():847] run started, returning control to user process
